e2e/content-services/document-list/document-list-component.e2e.ts

```diff
@@ -146,7 +146,7 @@ describe('Document List Component', () => {
         it('[C279929] Should be able to display the date with date type', async () => {
             await apiService.login(acsUser.username, acsUser.password);
             mediumDateUploadedNode = await uploadActions.uploadFile(mediumFileModel.location, mediumFileModel.name, '-my-');
-            const createdDate = moment(mediumDateUploadedNode.createdAt).format('ll');
+            const createdDate = format(new Date(mediumDateUploadedNode.createdAt), 'PPpp');
```

Comment:
'll' in moment corresponds to 'PP' in date-fns format, why did you have to make it 'PPpp', 'pp' is for time in 12:00:00 AM format, which 'll' does not include.

Reply:
changed format to 'PP' instead of 'PPpp'